From: Michael Biebl <biebl@debian.org>
Date: Tue, 13 Dec 2022 00:32:17 +0100
Subject: Skip flaky test_resolved_domain_restricted_dns in networkd-test.py

This test is part of DnsmasqClientTest and does not work reliably under
LXC/debci, so skip it for the time being.

Closes: #1025908
---
 test/networkd-test.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/networkd-test.py b/test/networkd-test.py
index 512137c..1bf1538 100755
--- a/test/networkd-test.py
+++ b/test/networkd-test.py
@@ -639,6 +639,7 @@ class DnsmasqClientTest(ClientTestBase, unittest.TestCase):
         with open(path) as f:
             sys.stdout.write('\n\n---- {} ----\n{}\n------\n\n'.format(os.path.basename(path), f.read()))
 
+    @unittest.skip("test is flaky:  https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1025908")
     def test_resolved_domain_restricted_dns(self):
         '''resolved: domain-restricted DNS servers'''
 
